enterprise/releases/2021-12-0.mdx

```diff
@@ -0,0 +1,72 @@
+
+## New features
+
+* Index Advisor
+  - This is a major new feature - you can read more about it in our blog posts:
+    - [A better way to index your Postgres database: pganalyze Index Advisor](/blog/introducing-pganalyze-index-advisor)
+    - [How we deconstructed the Postgres planner to find indexing opportunities](/blog/deconstructing-the-postgres-planner)
+* Single Sign-On with SAML, supporting Okta & others
+  - You can now enable Single Sign-On using the Integrations page (under Settings)
+  - Once enabled the main login page will ask users to login with SSO instead
+  - Note that existing users with conflicting email addresses will need to change their email or have their user removed before SSO logins are permitted
+* Add updateServer and deleteServer GraphQL mutations
+  - These can be used for automating updating the pganalyze configuration, e.g. to adjust the database hostname when using the local built-in collector
+* Track custom types for the Index Advisor
+* Track table column statistics (collected by ANALYZE) for the Index Advisor
+  - This only collects frequency and similar information, not actual table values
+  - Note this requires adding the special `get_column_stats` [helper function](https://github.com/pganalyze/collector#setting-up-a-restricted-monitoring-user) when using the restricted monitoring user
+* Update to [pganalyze-collector 0.41.2](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md)
+* Increase per-server table limit
+  - This adjusts the maximum permitted number of tables to 100,000 tables (from 5,000)
+  - Note that you may run into statistics processing timeouts on some systems when monitoring higher table counts
+* Show EXPLAIN plans on query detail graph
+  - This makes it easy to find outliers recorded via `auto_explain` or log-based EXPLAIN, directly in each query's runtime performance graph
+* Check-up: Group multiple references for an issue when appropriate
+  - e.g. this now groups together long-running idle transactions for the same role, or unused indexes on the same table
+
+
+## Bugfixes
+
+* Docker image: Use setpriv consistently for dropping privileges
+* Improve Enterprise license check error handling (e.g. when running into a firewall issue)
+* Enterprise self check: Don't test Redis connection if its built-in
+* Fix server issue links in e-mail notifications
+* Ignore explains with errors for weekly reports
+* Fix undefined region handling in database-level query performance summary
+* Shorten 'Queries Per Second' column heading to 'Queries / Second' to fix styling issue
+* Check-up: Fix coalescing almost-concurrent email issue notifications
+* Check-up: Avoid errors when role not present when processing connection issues
+* Improve query normalization of "GROUP BY" clauses
+* Query Details Graph: ignore clicks when no corresponding EXPLAIN found
```

Comment:
I think since showing EXPLAINs on the graph is new in this release, we can omit this.